server/src/BCAAPForm/bcaapform.entity.ts

```diff
@@ -0,0 +1,20 @@
+import { Entity, PrimaryGeneratedColumn, Column } from 'typeorm';
+import { CustomBaseEntity } from '../common/custom-base.entity';
+
+@Entity()
+export class BCAAPForm extends CustomBaseEntity {
```

Comment:
FormMetaData or FormSchema

Reply:
That makes way more sense, thanks Karthikk! Renamed to FormMetaData. Removed some unused files (Supporting-Documents) as well, made sure the app runs in Docker.
If everything is looking okay - ready to merge!